scipy/stats/_quantile.py

```diff
@@ -101,12 +101,27 @@ def _quantile_iv(x, p, method, axis, nan_policy, keepdims):
         p = xp.asarray(p, copy=True)
         p = xpx.at(p, p_mask).set(0.5)  # these get NaN-ed out at the end
 
-    return y, p, method, axis, nan_policy, keepdims, n, axis_none, ndim, p_mask, xp
+    if method != 'winsor' and options is not None:
+        message = "Only ``method='winsor'`` supports `options`."
+        raise ValueError(message)
+
+    options_out = None
+    if method == 'winsor':
+        options = {} if options is None else options
+        options_out = dict(round_left=xp.round, round_right=xp.round)
```

Comment:
Not sure what to call these keys. `round_left` sounds like "round to the left", but really it means "function to round with when `p < 0.5`".